x-pack/legacy/plugins/epm/server/lib/kibana/index_pattern/install.ts

```diff
@@ -99,28 +101,68 @@ export const dedupFields = (fields: Field[]) => {
 };
 
 export const transformField = (field: Field): IndexPatternField => {
-  const newField = { ...field };
+  const newField: IndexPatternField = {
+    name: field.name,
+    count: field.count ? field.count : 0,
+    scripted: false,
+    indexed: getVal(field.index, true),
```

Comment:
I assume `getVal` is from porting the golang code, but we can avoid the call if we use the nullish coalescing operator `??` like

```suggestion
    indexed: field.index ?? true,
```